README.md

```diff
@@ -12,9 +12,33 @@ For how to use the framework to author a Skill to be consumed, and creating an a
 
 ## Other Samples
 
-- [ObjectDetector](samples/ObjectDetector): detecting and classifying objects in images
-- [SkeletalDetector](samples/SkeletalDetector): estimating poses of people in images
-- [ConceptTagger](samples/ConceptTagger): obtaining classification scores of concepts in images
+### **[ObjectDetector](samples/ObjectDetector)**
+
+| ![ObjectDetector logo](./doc/ObjecDetectorLogo.png) | detecting and classifying objects in images |
+| -- | -- |
+| | |
```

Comment:
Why the extra rows? Same for skills below

Reply:
only way to create a 1-row table :\

Reply:
resolved